### PR TITLE
Cisco Umbrella Investigate - fix regex for email validation in the get-domains-for-email-registrar command

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/30734

## Description
fixed regex for email address validation 

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests
See failing test in https://app.circleci.com/pipelines/github/demisto/content/44855/workflows/9ce91f04-ec48-4fd7-ad0f-c3799f6daf08/jobs/192982

### CODE REVIEW

File: Packs/Cisco-umbrella/Integrations/Cisco-umbrella-investigate/Cisco-umbrella-investigate.py [hunk 1]
```diff
@@ -1150,7 +1150,7 @@ def get_domains_for_email_registrar_command():
         if sort != 'created' and sort != 'updated':
             return_error('The parameter sort accept only these values: created/updated/expired.')
     for email in emails:
-        if re.match('^[_a-z0-9-]+(\.[_a-z0-9-]+)*@[a-z0-9-]+(\.[a-z0-9-]+)*(\.[a-z]{2,4})$', email) is None:
+        if not re.match(emailRegex, email):
```
Comment (ShahafBenYakir) [inline Packs/Cisco-umbrella/Integrations/Cisco-umbrella-investigate/Cisco-umbrella-investigate.py:1153]: `emailRegex` where is this from?

**Comment by ShahafBenYakir** [comment]
Please share a failing unit-test and a successful one.

File: Packs/Cisco-umbrella/Integrations/Cisco-umbrella-investigate/Cisco-umbrella-investigate.py (hunk 1 above)
Comment (Itay4) [reply Packs/Cisco-umbrella/Integrations/Cisco-umbrella-investigate/Cisco-umbrella-investigate.py:1153]: commonserverpython

**Comment by Itay4** [comment]
> Please share a failing unit-test and a successful one.

look at the comment, there is a failing test playbook there

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**